Index: app/src/main/java/com/onquantum/utaxi/wizard/FragmentWizardStepOne.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/onquantum/utaxi/wizard/FragmentWizardStepOne.java	(date 1412893679000)
+++ app/src/main/java/com/onquantum/utaxi/wizard/FragmentWizardStepOne.java	(revision )
@@ -20,6 +20,7 @@
 import com.google.android.gms.maps.MapFragment;
 import com.google.android.gms.maps.model.CameraPosition;
 import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
 import com.onquantum.utaxi.R;
 import com.onquantum.utaxi.services.TrackingService;
 
@@ -45,9 +46,12 @@
                 @Override
                 public void onCameraChange(CameraPosition cameraPosition) {
                     Log.i("info"," Camera Position = " + cameraPosition.toString());
+                    googleMap.clear();
+                    googleMap.addMarker(new MarkerOptions().position(cameraPosition.target));
                 }
             });
         }
+
         ((Button) root.findViewById(R.id.button)).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
